modules/apps/headless/headless-admin-content/headless-admin-content-impl/src/main/java/com/liferay/headless/admin/content/internal/resource/v1_0/StructuredContentResourceImpl.java

```diff
@@ -197,6 +230,90 @@ public EntityModel getEntityModel(MultivaluedMap multivaluedMap) {
 				this::_toExtensionStructuredContent));
 	}
 
+	@Override
+	public StructuredContent postSiteStructuredContentDraft(
+			@NotNull Long siteId, StructuredContent structuredContent)
+		throws Exception {
+
+		DDMStructure ddmStructure = DDMStructureUtil.getDDMStructure(
+			_ddmStructureService, structuredContent);
+
+		LocalDateTime localDateTime = LocalDateTimeUtil.toLocalDateTime(
```

Comment:
Variables are defined in the order they are used and you don't need localDateTime here, I think this should be right before servicecontext

Reply:
Done!